DiscordBot/permissions.py

```diff
@@ -0,0 +1,708 @@
+# Aside: At the time of writing, there is no set_voice_channel_status
+# permission available for categories and voice channels in the discord.py API.
+# The pull request for the permission is being worked on here:
+# https://github.com/discord/discord-api-docs/pull/6400
+# P.S.: create_events is not supported for voice channels, stage channels
+# and categories either.
+
+# Channel Ordering:
+# Note that forum, announcement, and stage channels are options only for community channels.
+# Text-based channels such as text, forum, and announcement channels can swap orders.
+# Voice-based channels such as voice and stage channels can swap orders.
+# However, text-based and voice-based channels cannot swap orders;
+# moreover, voice-based channels must be ordered below text-based channels
+# in each category and when not in a category.
```

Comment:
what are these comments?

Reply:
These comments are just so that I remember the way in which Discord orders different types of channels.